app/posts/manage-teacher-training-applications/2021-08-10-helping-users-check-how-quickly-courses-are-filling-up.md

```diff
@@ -1,22 +1,20 @@
 ---
 title: Helping users check how quickly courses are filling up
-description: Various changes to get the design of the reports area ready for an initial release
+description: Various small changes to get the design of the reports area ready for an initial release
 date: 2021-08-10
 tags:
-- MN027
+  - MN027
 screenshots:
   items:
     - text: Reports landing page
-      src: reports.png
+      src: reports-landing-page--single-organisation.png
     - text: Reports landing page when the user belongs to multiple organisations
-      src: reports--multi-org.png
-    - text: Data export page now without a link to the HESA export
-      src: export-application-data.png
-    - text: HESA export now with a new breadcrumb
-      src: export-hesa-data.png
+      src: reports-landing-page--multiple-organisations.png
+    - text: Status of active applications report
+      src: status-of-active-applications-report.png
 ---
 
-We’ve made some changes to the existing ‘Export data’ area, as part of the work to help users make the application process better.
+We’ve made some changes to the reports section of the service, following on from work to [provide status and progress reports alongside data exports](/manage-teacher-training-applications/providing-status-and-progress-reports-alongside-data-exports/).
```

Comment:
Is there anywhere we reference the overarching "help users make the application process better" theme?